packages/editor-ui/src/components/WorkflowHistory/WorkflowHistoryListItem.vue

```diff
@@ -75,6 +77,8 @@ onMounted(() => {
 		offsetTop: itemElement.value?.offsetTop ?? 0,
 		isActive: props.isActive,
 	});
+	isAuthorElementTruncated.value =
```

Comment:
If there is a single author with long first name, the truncation will only display the first name truncated?

Reply:
The span element which contains the first author full name (and the `+ :number` annotation if there are more) is truncated with CSS and the tooltip is visible in the following scenarios:
1. There are more than one author in the version
2. The first displayed author name is too long